content/apps/using-oauth2.md

```diff
@@ -0,0 +1,157 @@
+# Using OAuth 2.0 to connect an App
+
+The PIM uses OAuth 2.0 to manage the authorization of Apps.
+
+![App activation diagram](../img/apps/app-activation-sequence-diagram.png)
+
+
+## Credentials
+
+To begin, you must obtain valid OAuth 2.0 client credentials by registering your App on the
+[Akeneo Marketplace](https://marketplace.akeneo.com/node/add/extension).
```

Comment:
```suggestion
The client credential is a pair of client id and client secret.
```
You mention the client_id later but you don't explain where it comes from. Should be clearer if you refer it here.

Reply:
Do you think it needs to be clarified here ?
When you ask for client credentials on the marketplace, you have a huge screen saying:
"Here is your client_id and client_secret, keep them."

Reply:
Also, any OAuth library ask for those `client_id` and `client_secret`

Reply:
It's just a sentence to be sure we are the clearer as possible. I would add it.

Reply:
Ok.

@meganelepalud-akeneo, do you want to check the wording of the following new sentence ?
```
The client credentials are composed of a pair of `client_id` and `client_secret`.
```